wpeverest/user-registration
# Usage tracking implementation

### All Submissions:






### Changes proposed in this Pull Request:



Usage tracking implementation on misc settings.

### How to test the changes in this Pull Request:

1. First go to Install & activate the plugin
2. Go to misc settings 
3. Enable allow usage tracking
4. Check if data is sent to the stats server correctly. First time on setting update and after then on every 15 days. 

### Types of changes:




### Other information:




### Changelog entry

> Added - Usage tracking implementation after user consent.
